components/Navbar.js

```diff
@@ -3,49 +3,68 @@ import React from 'react';
-        </span>
-      </div>
-      <div className="flex-1 px-2 mx-2 hidden md:flex">
-        <div className="items-stretch hidden lg:flex">
-          <Link href="/">
-            <a className="btn btn-ghost btn-sm rounded-btn">{'Home'}</a>
-          </Link>
-          <Link href="/in-kind">
-            <a className="btn btn-ghost btn-sm rounded-btn">{'In-Kind'}</a>
-          </Link>
-          <Link href="/fund">
-            <a className="btn btn-ghost btn-sm rounded-btn">{'Fund'}</a>
-          </Link>
-          <Link href="/give-your-time">
-            <a className="btn btn-ghost btn-sm rounded-btn">
-              {'Give Your Time'}
-            </a>
-          </Link>
-          <Link href="/partner-with-us">
-            <a className="btn btn-ghost btn-sm rounded-btn">
-              {'Partner With Us'}
-            </a>
-          </Link>
-        </div>
-      </div>
-      <div className="flex-none hidden sm:flex">
-        {'Help Families in Need'}
-        <button className="btn btn-square btn-ghost" type="button">
-          <FaRegHeart className="inline-block w-5 h-5 stroke-current" />
-        </button>
-      </div>
-      <div className="flex-none hidden sm:flex">
-        <button className="btn btn-square btn-ghost" type="button">
-          <BsSearch className="inline-block w-5 h-5 stroke-current" />
+    <>
+      <SideNavbar setShow={setShowSideNav} show={showSideNav} />
+      <div className="navbar mb-2 shadow-lg bg-neutral text-neutral-content justify-center sticky top-0 z-50">
+        <button
+          className=" btn-ghost flex  lg:hidden"
+          onClick={() => setShowSideNav(!showSideNav)}
+          type="button"
+        >
+          {showSideNav ? (
+            <ImCross className="inline-block w-5 h-5 stroke-current " />
+          ) : (
+            <GiHamburgerMenu className="inline-block w-5 h-5 stroke-current " />
+          )}
         </button>
+        <div className="flex-none px-2 mx-2">
+          <span className="text-lg font-bold">
+            {'Margarita Humanitarian Foundation'}
+          </span>
+        </div>
+        <div className="flex-1 px-2 mx-2 hidden md:flex">
+          <div className="items-stretch hidden lg:flex">
+            <Link href="/">
+              <a className="btn btn-ghost btn-sm rounded-btn">{'Home'}</a>
+            </Link>
+            <Link href="/in-kind">
+              <a className="btn btn-ghost btn-sm rounded-btn">{'In-Kind'}</a>
+            </Link>
+            <Link href="/fund">
+              <a className="btn btn-ghost btn-sm rounded-btn">{'Fund'}</a>
+            </Link>
+            <Link href="/give-your-time">
+              <a className="btn btn-ghost btn-sm rounded-btn">
+                {'Give Your Time'}
+              </a>
+            </Link>
+            <Link href="/partner-with-us">
+              <a className="btn btn-ghost btn-sm rounded-btn">
+                {'Partner With Us'}
+              </a>
+            </Link>
```

Comment:
I noticed you fixed the responsiveness issue here with the extra space on the side. Nice! If you can, it would be nice to fix the overlapping text here at this size in this image.
<img width="800" alt="Screen Shot 2021-10-04 at 4 32 59 PM" src="https://user-images.githubusercontent.com/68655112/135939323-c7576202-3091-4530-b579-4d996af88d06.png">

Reply:
Hi, can you tell me where is navbar class is writtern? 
referring to line 16 `Navbar.js`